tests/apollo/test_skvbc_backup_restore.py

```diff
@@ -59,16 +59,19 @@ class SkvbcBackupRestoreTest(ApolloTest):
     @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7)
     async def test_checkpoint_propagation_after_restarting_replicas(self, bft_network):
         """
-        Here we trigger a checkpoint, restart all replicas in a random order with 5s delay in-between,
-        both while stopping and starting. We verify checkpoint persisted upon restart and then trigger
+        Here we trigger a checkpoint, restart all replicas.
+        We verify checkpoint persisted upon restart and then trigger
         another checkpoint. We make sure checkpoint is propagated to all the replicas.
         1) Given a BFT network, we make sure all nodes are up
         2) Send sufficient number of client requests to trigger checkpoint protocol
-        3) Stop all replicas in a random order (with 5s delay in between)
-        4) Start all replicas in a random order (with 5s delay in between)
+        3) Stop all replicas in a random order
+        4) Start all replicas in a random order
         5) Make sure the initial view is stable
         6) Send sufficient number of client requests to trigger another checkpoint
         7) Make sure checkpoint propagates to all the replicas
+
+        Note: UDP configuration waits for 5 seconds until it assumes network communication is established.
+        A replica can thus trigger a view change if it
```

Comment:
`A replica can thus trigger a view change if it`, if it what?

Reply:
done